src/sinks/datadog/traces/tests.rs

```diff
@@ -23,6 +24,10 @@ mod dd_proto {
     include!(concat!(env!("OUT_DIR"), "/dd_trace.rs"));
 }
 
+mod ddsketch_full {
+    include!(concat!(env!("OUT_DIR"), "/ddsketch_full.rs"));
+}
```

Comment:
Would it make sense to share this as a module rather than including it twice, here and in `src/sinks/datadog/traces/stats.rs`? Or just including it by `super::traces::ddsketch_full`

Reply:
I agree, will do the same with the .proto for traces.